syftlib/src/main/java/org/openmined/syft/networking/clients/SocketClient.kt

```diff
@@ -32,6 +32,12 @@ import java.util.concurrent.atomic.AtomicBoolean
 
 private const val TAG = "SocketClient"
 
+/**
+ * Used to communicate and exchange data throw web socket with PyGrid
+ * @param syftWebSocket Create web socket connection
```

Comment:
@property tags would be better here I guess

Reply:
Do you mean using @property for the constructor params? 
I guess it's not the best practice, I tried to change it and Android Studio doesn't recognize the params any more.

Reply:
Hmm that's wierd. It shouldn't happen.
On my setup @property name works quite well. In fact this is the recommended way for describing constructor properties. 

https://kotlinlang.org/docs/reference/kotlin-doc.html

Reply:
Oh, I was checking the javadoc, but I've noticed something `@property` works well on SocketClient since the constructor params declared with `val`, and on SyftWebSocket it doesn't

Reply:
Yeah without the val it isn't a class property anymore ao that may be the reason